subprojects/docs/src/docs/userguide/build_init_plugin.adoc

```diff
@@ -76,12 +76,12 @@ The conversion process has the following features:
 * Supports both single module and multimodule projects
 * Supports custom module names (that differ from directory names)
 * Generates general metadata - id, description and version
-* Applies maven, java and war plugins (as needed)
+* Applies <<publishing_maven,maven-publish>>, <<java_plugin,java>> and <<war_plugin,war>> plugins (as needed)
```

Comment:
Please use "Maven Publish, Java and War Plugins". I realise that the original text used lower case, but this would be a good opportunity to change that.

Reply:
Done